frontend/src/Pages/User.tsx

```diff
@@ -65,6 +66,9 @@ function User() {
                 My Wallet
               </Button>
             </Box>
+            <Box>
```

Comment:
Add a margin to keep things nice and tidy. Spaced out, but not too spaced out! Here is my suggestion:

`<Box sx={{ marginTop: 5 }}>`

Reply:
Additionally, lets add a label for the data table so the user knows what they're looking at. Right before the `<StockPortfolioComponent />` we could add a title like this:

```
      <Typography variant="h5" align="left" marginBottom={2}>
        Your portfolio:
      </Typography>
```

This should keep everything nice and organized. Keep up the great work!